app/src/main/java/backgammon/Movelet.java

```diff
@@ -2,6 +2,7 @@
 
 public record Movelet(int from, int to, int die) {
 
+
```

Comment:
Make sure that when you are making a pull request that you don't add additional newlines inbetween the methods.
Change your IDE's text font size so that you can see the methods more clearly.

Reply:
I'll fix them for now but remember that for the next PR you make